src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -76,6 +77,23 @@ public TracerSettings(IConfigurationSource source)
                                false;
 
             Integrations = new IntegrationSettingsCollection(source);
+
+            if (source == null)
+            {
+                GlobalTags = new Dictionary<string, string>();
+            }
+            else
+            {
+                var gTags = source.GetString(ConfigurationKeys.GlobalTags);
+                if (gTags != null)
+                {
+                    GlobalTags = JsonConvert.DeserializeObject<Dictionary<string, string>>(gTags);
```

Comment:
Are we using JSON as the format for this setting? I think we should use something simpler, like `key1:value1,key2:value2`. It probably also a good idea to make this consistent with other Tracers.

Reply:
I'll look again at using a `NameValueCollection`. I had a problem converting it earlier but that should work.

Reply:
+1. Comma-separated or semicolon-separated values would be easier to work with, and if we can standardize it with other Tracers that would be awesome.

Reply:
It looks like the other Tracers support `tag`:`value` with comma-separators. I can do that to be consistent.

Reply:
Although the comma-separate, tag:value pairs with a colon separator is what I see in most of the other tracers, it is not a common format you see in .NET.

Reply:
Implemented csv with colon-separated key and value.